ixofoundation/ixo-webclient
# chore: add progress bar

## Scope
https://nonaredteam.atlassian.net/browse/IXO-712

## Work Done
Added progress indicator UI
Added Redux actions to allow the user to skip to any step

## Steps to Test
Run the app and go to submit a claim
Check that the Progress bar appears as it should as per the designs
Check that it increments and decrements when you go through the steps

## Screenshots
![image](https://user-images.githubusercontent.com/46312800/85832329-27da1a00-b790-11ea-9ac7-29a9f1e40b54.png)
